posts/using-napari-for-cosmx-data/index.qmd

```diff
@@ -0,0 +1,525 @@
+limits. Protein results are also viewed as an image layer in napari. For CosMx 
+RNA experiments, the detected transcripts are viewed as a points layer in napari.
+Cell segmentation results are displayed as an image layer of the cell boundaries.
+The cell shapes can also be colored by metadata such as cell type. @fig-example-gif
+shows some examples of these in an animated format. 
+
+```{r}
+#| eval: true
+#| echo: false
+#| label: "fig-example-gif"
+#| fig-cap: "Example animation made with `napari-cosmx` showing ligand-receoptor interactions in a healthy prostate sampele. Cell types fill in the cell boundaries. When cells are transparent, one can see more easily see the spatial location of _S100A8_, _S100A9_ ligand RNA transcripts with the _TLR4_ receptor transcript. While animations are certainly not needed for all purposes, this one highlights that the plugin can color RNA transcripts (points layer), cell-level metadata like cell types, and cell boundaries. It can also visualize IF (not shown here; see @fig-pancreas for example of IF staining)." 
+
+knitr::include_graphics("./figures/prostate.gif")
+```
+
+Beyond the basic interactivity and viewing of napari, other posts in this series
+will provide examples of tips and tricks as well as more advanced analysis. See the 
+[series topics](https://nanostring-biostats.github.io/CosMx-Analysis-Scratch-Space/about.html) for what's coming ahead!
+
+
+# Using the `napari-cosmx` plugin 
+
+
+::: {.callout-note}
+While napari and the `napari-cosmx` can be installed on many systems, CosMx data
+can be quite large. A slide with a large number of FOVs may very well exceed the
+capabilities of a standard laptop.
+:::
+
+
+## Installing `napari-cosmx` {#sec-installing}
+
+The specific download instructions depend on your operating system (see tabs below)
+but the general procedure is the same: 1) installing napari 0.4.17 and 2) installing the `napari-cosmx` plugin
+
+::: {.callout-note}
+The `napari-cosmx` plugin was developed with napari 0.4.17. There are some breaking
+changes that we have noticed if using the plugin with newer version of napari (_e.g._,
+0.4.18). At the time of writing this post, please make sure to install version 0.4.17. 
+
+:::
+
+::: {.panel-tabset}
+
+### Windows Install
+
+::: {.callout-note}
+Depending on your browser and security settings you may get warnings when downloading or running some of the links below.
+:::
+
+#### Part 1: Installing Napari
+
+:::: {.columns}
+
+::: {.column width="45%"}
+
+The napari project contains platform-specific bundled apps for each release that
+don’t require you to first install a Python environment. You simply run the installer
+and a link will be added to your Start Menu as with a typical app installation. 
+The napari-CosMx plugin currently expects napari 0.4.17.
+
+[Click to download the Windows Installer](https://github.com/napari/napari/releases/download/v0.4.17/napari-0.4.17-Windows-x86_64.exe)
```

Comment:
Can you add link to the github page with this exe pointed out, similar to what's done for Mac, as an alternative?

Reply:
Not easily. That's because the assests section in napari isn't anchored so I can't reference it easily.